### ISSUE TITLE
[1.10.2] SpongeForge 1675 cannot find WE(forge) block changes

### ISSUE THREAD

**Issue by Rasgnarok** [SpongePowered/SpongeForge#854]
So, at this point, it is getting too hard to actually use the plugin without coming into this issue, that spams the console for a solid 5 minutes and makes everyone disconnect.

I am running:
```
worldedit-forge-mc1.10-6.1.4-SNAPSHOT-dist.jar
spongeforge-1.10.2-2052-5.0.0-BETA-1675.jar
forge-1.10.2-12.18.1.2052-universal.jar
Polis-1.10.2-2.6.8.jar
```

If anyone understands what is going on, you are very, very welcome. I have seen the error across sf versions _(after versions 1629)_, meaning this isn't just one version's problem. It seems to do this every time there are commands that cause a block change in fluids - namely, lava, water.

Here's the error log: http://pastebin.com/fCPbcgjp

**Comment by FiveEYZ** [comment]
have you tried the worldedit made for sponge?

**Comment by LotuxPunk** [comment]
@FiveEYZ Do you have a link ?

**Comment by Rasgnarok** [comment]
No, since this very same WE is fine without spongeforge - I tested on singleplayer. Meaning this is a spongeforge issue.

**Comment by FiveEYZ** [comment]
@Rasgnarok what do u mean? ofc it works fine without SF, cause it was built only against Forge

They also have a jar built against sponge and here is the link to the one im using:
http://builds.enginehub.org/job/worldedit/9352

then take the one named ` worldedit-sponge-mc1.10.2-6.1.4-SNAPSHOT-dist.jar`
this one gets updated to reflect the changes made to sponge

here ya go @LotuxPunk

**Comment by Rasgnarok** [comment]
I know, but the ForgeWE should be compatible with Spongeforge. Trying a sponge version means that spongeforge is no longer made to be compatible with core forge plugins.

Hence why I am reporting a bug that needs fixing - only if it is impossible to fix should you ever jump plugins because of a bug.

**Comment by LotuxPunk** [comment]
Thank you @FiveEYZ !

**Comment by wizjany** [comment]
Do you happen to be running in //fast mode?

**Comment by Rasgnarok** [comment]
Did not know that was an option - no I am not.

**Comment by gabizou** [comment]
What did you use to perform the world changes? From what it looks like, it's just a chat event getting executed, not a command, which is why Sponge is flipping out because it's not exactly expected for world block modifications to take place due to a chat event.

**Comment by Rasgnarok** [comment]
Well, WorldEdit works with commands, say //set water (having selected with the wand stone). I am unsure if that is what you are referring to - nothing more needs to be done to reproduce the bug. The usage of the double slash here is default-worldedit.

As soon as the command will change water/lava, console flips out.

**Comment by bloodmc** [comment]
@Rasgnarok The latest build should fix this. Please confirm.

**Comment by hunter688** [comment]
I think issue is resolved, tested similar #858 with latest SF now.
